src/crypto/TS.h

```diff
@@ -23,6 +23,8 @@
 #include <memory>
 
 typedef struct pkcs7_st PKCS7;
+typedef struct CMS_ContentInfo_st CMS_ContentInfo;
+typedef struct TS_tst_info_st TS_TST_INFO;
```

Comment:
I don't want pollute other code with openssl includes.
It C++ standard practice to forward declare classes and structs in headers.